ci/cpu/upload-anaconda.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+#
+# Adopted from https://github.com/tmcdonell/travis-scripts/blob/dfaac280ac2082cd6bcaba3217428347899f2975/update-accelerate-buildbot.sh
+
+set -e
+
+export UPLOADFILE=`conda build conda/recipes/dask-cuda --python=$PYTHON --output`
+CUDA_REL=${CUDA:0:3}
+if [ "${CUDA:0:2}" == '10' ]; then
+  # CUDA 10 release
+  CUDA_REL=${CUDA:0:4}
+fi
+
+SOURCE_BRANCH=master
+
+if [ "$LABEL_MAIN" == "1" -a "$BUILD_ABI" == "1" ]; then
+  LABEL_OPTION="--label main --label cuda${CUDA_REL}"
+elif [ "$LABEL_MAIN" == "0" -a "$BUILD_ABI" == "1" ]; then
+  LABEL_OPTION="--label dev --label cuda${CUDA_REL}"
+elif [ "$LABEL_MAIN" == "1" -a "$BUILD_ABI" == "0" ]; then
+  LABEL_OPTION="--label cf201901 --label cf201901-cuda${CUDA_REL}"
```

Comment:
does `cf201901` have a particular significance?

Reply:
That's the label for ABI=OFF, but we don't support that anymore, so I'll remove this actually.